server/src/test/proto/spine/test/event/events.proto

```diff
@@ -41,6 +41,12 @@ message ProjectCreated {
     }
 }
 
+message ProjectCreatedSeparateEnrichment {
+    option (enrichment_for) = "ProjectCreated";
+
+    string project_name = 1 [(by) = "project_id"];
```

Comment:
Please have a test data where we have same package type name right in the `by` specification.